package.json

```
@@ -17,7 +17,8 @@
         "prettier": "prettier -c .",
         "prettier:fix": "prettier -w .",
         "commit": "cz",
-        "precommit": "lint-staged"
+        "precommit": "lint-staged",
+        "prepare": "husky install"
```

Can you please add `is-ci` package to prevent hooks installation on ci envs?
 
```suggestion
        "prepare": "is-ci || husky install"
```

---

Reflected in d2077c3c7d0158f67837eda6e8400712ad2b4c7e